### PR TITLE
[technical] budgeting/assests/*, PlansList: update linting rules and jsx files

### PR DESCRIPTION
**Pull request by phillimorland**

### CODE REVIEW

File: meinberlin/apps/plans/assets/PlansList.jsx [hunk 1]
```diff
@@ -66,8 +66,9 @@ class LazyBackground extends React.Component {
         className={this.props.isHorizontal ? 'u-lg-only-display maplist-item__img' : 'maplist-item__img'} style={{
           backgroundPosition: 'center',
           backgroundRepeat: 'no-repeat',
-          backgroundImage: this.state.source ? `url(${this.state.source})` : 'none'
-        }} alt=""
+          backgroundImage: this.state.source ? url(this.state.source) : 'none'
```
Comment (phillimorland) [inline meinberlin/apps/plans/assets/PlansList.jsx:69]: @goapunk i'm bad a syntax, url is not defined? sorry dumb question

Comment (goapunk) [reply meinberlin/apps/plans/assets/PlansList.jsx:69]: needs to be `'url(' + this.state.source + ')'` I guess?

File: meinberlin/apps/budgeting/assets/BudgetingProposalListItem.jsx [hunk 2]
```diff
@@ -12,14 +12,14 @@ export const BudgetingProposalListItem = (props) => {
   const { proposal, permissions, tokenvoteApiUrl } = props
   const safeLocale = props.locale ? props.locale : undefined
   const date = proposal.modified
-    ? `${updatedOnStr} ${toLocaleDate(
+    ? { updatedOnStr } + toLocaleDate(
```
Comment (goapunk) [inline meinberlin/apps/budgeting/assets/BudgetingProposalListItem.jsx:15]: the {} are not needed, also it is missing a whitespace between `updatedOnStr` and `toLocaleDate` I think. So proper way would be `updatedOnStr + ' ' + toLocaleDate(`

File: meinberlin/apps/budgeting/assets/ControlBarDropdown.jsx [hunk 3]
```diff
@@ -47,14 +47,14 @@ export const ControlBarDropdown = props => {
         aria-expanded="false"
         id={props.filterId}
       >
-        {`${filter.label}: ${currentChoiceName}`}
+        {filter.label + ':' + currentChoiceName}
```
Comment (goapunk) [inline meinberlin/apps/budgeting/assets/ControlBarDropdown.jsx:50]: missing whitespace after the : ?

File: meinberlin/apps/budgeting/assets/BudgetingProposalList.jsx [hunk 4]
```diff
@@ -45,7 +49,7 @@ export const BudgetingProposalList = (props) => {
         <ul className="u-list-reset">
           {data.map((proposal, idx) =>
             <BudgetingProposalListItem
-              key={`budgeting-proposal-${idx}`}
+              key={'budgeting-proposal-' + idx}
```
Comment (goapunk) [inline meinberlin/apps/budgeting/assets/BudgetingProposalList.jsx:52]: remove the whitespace before the +

File: meinberlin/apps/budgeting/assets/BudgetingProposalListItem.jsx [hunk 5]
```diff
@@ -12,14 +12,14 @@ export const BudgetingProposalListItem = (props) => {
...
         proposal.modified,
         safeLocale
-      )}`
-    : `${createdOnStr} ${toLocaleDate(
+      )
+    : { createdOnStr } + toLocaleDate(
```
Comment (goapunk) [inline meinberlin/apps/budgeting/assets/BudgetingProposalListItem.jsx:19]: same

File: meinberlin/apps/budgeting/assets/ListItemBadges.jsx [hunk 6]
```diff
@@ -14,8 +14,8 @@ export const ListItemBadges = props => {
     if (badge[0] !== 'moderator_feedback') {
       return labelClass
     }
-    const modTypeClass = `label--${badge[2].toLowerCase()}`
-    return `${labelClass} ${modTypeClass}`
+    const modTypeClass = 'label--' + badge[2].toLowerCase()
+    return { labelClass } + { modTypeClass }
```
Comment (goapunk) [inline meinberlin/apps/budgeting/assets/ListItemBadges.jsx:18]: mssing whitespace between the two and redundant brackets

File: meinberlin/apps/budgeting/assets/helpers.js [hunk 7]
```diff
@@ -15,5 +15,5 @@ export const toLocaleDate = (isodate, locale = 'de-DE') => {
 // input: "0"
 // output: " 0 "
 export const wrapSpaces = (value) => {
-  return ` ${value} `
+  return { value }
```
Comment (goapunk) [inline meinberlin/apps/budgeting/assets/helpers.js:18]: missing whitespace at start and end of word and redundant brackets

**Review by fuzzylogic2000 — APPROVED** [review]
Supercool!

**Pull request merged**